### PR TITLE
16aug template2

### PR DESCRIPTION
**Pull request by cgbuen**
Update templates

### CODE REVIEW

File: docs/moov_stdlib/global.html [hunk 1]
```diff
@@ -246,6 +246,64 @@ <h3 class="subsection-title">Members</h3>
...
+    
+
+    
+
+    
+
+    
+
+    
+
```
Comment (cin210) [inline docs/moov_stdlib/global.html:291]: what're all these newlines about?

Comment (cin210) [inline docs/moov_stdlib/global.html:291]: Is this a result of all the commented code in the js files?

File: tmpl/tmpl/layout.tmpl [hunk 2]
```diff
@@ -17,7 +17,7 @@
 <br class="clear">
 
 <footer>
-    Documentation generated at <?js= (new Date()) ?>
+    <i>Last updated <?js= new Date() ?></i>
```
Comment (cin210) [inline tmpl/tmpl/layout.tmpl:20]: Good change

File: docs/moov_stdlib/global.html (hunk 1 above)
Comment (cgbuen) [reply docs/moov_stdlib/global.html:291]: @cin210 Looks like compiling the extra whitespace has sort of always been a part of the framework: https://github.com/moovweb/moov_docs/blob/c1e90bc9afee3a379b92c0a7dd61d2c5146ef609/docs/moov_cheerio/global.html

**Pull request merged**